source/contracts/reporting/ReportingWindow.sol

```diff
@@ -327,6 +351,16 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable, IReportingWi
         return registrationToken == _shadyRegistrationToken;
     }
 
+    function isContainerForReportingToken(IReportingToken _shadyReportingToken) public afterInitialized constant returns (bool) {
```

Comment:
Elsewhere I changed a bunch of these to take in a `Typed` as a parameter, then after type checking I cast to the expected type.  It looks like you already have the type check, but since this is taking in an `IReportingToken` it means the caller must already know the correct type.  Consider changing this to take in a `Typed` like the others (I think Universe is where I updated them all).  This change is really a callsite convenience thing, feel free to not do it if you think it hurts more than helps.  My only argument is consistence.

Same note for the others in this file.

Reply:
Will change to Typed